getFeatureFlagById/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"log"
+	"net/http"
+)
+
+func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	id, found := req.PathParameters["flagId"]
+
+	if !found {
+		log.Println("flagId not passed")
+		utils.ClientError(http.StatusBadRequest, "Flag ID not passed in request")
+	}
+	featureFlag, err := database.ProcessGetFeatureFlagByHashKey(utils.FeatureFlagAttributes["Id"],id)
+	if err != nil {
+		utils.ServerError(err)
+	}
+
+	log.Println(featureFlag, " is the feature flag")
```

Comment:
won't logging this after checking if it is nil make more sense?

Reply:
Correct, I've added for debugging, will move it after the check